src/main/scala-2.13+/scala/concurrent/stm/compat/package.scala

```diff
@@ -0,0 +1,10 @@
+package scala.concurrent.stm
+
+import scala.collection.mutable
+
+package object compat {
+  implicit class MutableMapExtensionMethods[A, B](val map: mutable.Map[A, B]) extends AnyVal {
+    def mapValuesInPlace(f: (A, B) => B): map.type =
+      map.mapInPlace{ case (a, b) => (a, f(a, b)) }
+  }
```

Comment:
You should add a comment saying that this is temporary and should be removed for 2.13.0-M5, which will provide the `mapValuesInPlace` operation.